plugins/title.js

```diff
@@ -37,6 +37,15 @@ function extractYoutubeTitle(match, callback) {
   })
 }
 
+function extractGithubTitle(match, callback) {
+  getPage(match, function(body) {
+    var $ = cheerio.load(body)
+    var title = $('.repository-meta-content > span:nth-child(1)').text().trim()
```

Comment:
`span:nth-child(1)` is a bit too much guessing for my taste, I think using the github API is a way better approach :)